### DISCUSSION TITLE
Selecting a point on the map to trim a track

### DISCUSSION THREAD

**Discussion by madduck** [osmandapp/OsmAnd#20693] [General]
Hello,

We're on a bike tour along a GPX track that's 500km long and composed of roughly 10,000 points. We're going to the next town tomorrow, nowhere near the end of the track. It seems I have three options only:

1. Get routed to the next town by OSM internally, which ends up ignoring the track;
2. Choose to "follow the track", then add the next town as first intermediate destination. This unfortuately means that the distance and time to goal will display the track's final ending location, and also graphs like the elevation profile show the whole track, even though I'll only be interested in the segment from here to the next town this time;
3. Edit the track, i.e. select the current and destination point and split or trim the track accordingly.

1–2 are suboptimal, and I'd like to go with the third option. However, it seems not possible to get to a point's context menu from the map, only from the list of all points. And with 10,000 points, scrolling that list becomes really slow already after 300 points, plus there is no easy way for me to find out which point I actually need to select.

How can I edit the track and select a point visually, then enter its context menu so I can split the track there?

Or is there another way to achieve "route me to this destination, along this track, but not further"?

Thank you,
martin

**Comment by vshcherb** [comment]
You can select point in Plan Route - Edit and there is a button Trim before / after

**Reply by madduck** [reply]
I know you can select the point in the list, but as I wrote, the list is impossible to navigate and it's also not really easy to identify just which point to select/edit...

**Reply by madduck** [reply]
to be clear, I want to visually select a point on the map and edit it/enter the context menu for it.

**Reply by vshcherb** [reply]
Yes just select it on the map. You need to zoom in and see the points and click on it - https://docs.osmand.net/docs/user/plan-route/create-route/#point-context-menu

**Reply by vshcherb** [reply]
I created issue cause points appear only on very high zoom strangely - https://github.com/osmandapp/OsmAnd/issues/20722. But they appear!

**Comment by sjvudp** [comment]
It seems to me what you really want is to split the track into multiple tracks (day tours) before starting.
Maybe a function to split a track automatically into specific time intervals or distances could be added.
Also a "combine tracks" could be interesting.
